solutions/usaco-712.mdx

```diff
@@ -160,7 +160,72 @@ For every cow (say cow $N$), we can check which cows have one entry/exit point b
 **Time Complexity:** $\mathcal{O}(T^3)$, where $T$ represents the total number of characters in Farmer John's string.
 
 <LanguageSection>
-	
+
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+
```

Comment:
```suggestion
using namespace std;

```

Reply:
hey op, all of these changes are optional, if you wanna qualify the namespace each time it's fine by me

Reply:
f these changes are optional, if you wanna qualify the namespace each time it's fine by me



> hey op, all of these changes are optional, if you wanna qualify the namespace each time it's fine by me

i think it's ok to keep them.